_events/2024-0806-2-16-release-meetings.markdown

```diff
@@ -0,0 +1,60 @@
+---
+calendar_date: '2024-08-06'
+eventdate: 2024-08-06 09:00:00 -0700
```

Comment:
If the 1st go/nogo meeting is on 08/02, you should update this date to reflect on the calendar.

Reply:
I see so the `eventdate` and `calendar_date:` is not the release date ?

Reply:
It will change over time as events happens and eventually land on the actual release date.

Reply:
Means you will need to keep sending PRs to change this as event goes.

Reply:
Thanks @peterzhuamazon, updated the PR.